<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpenGL_Form</class>
 <widget class="QWidget" name="OpenGL_Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>773</width>
    <height>730</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QSlider" name="points_amount_Slider">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>30</y>
     <width>16</width>
     <height>471</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100000</number>
   </property>
   <property name="singleStep">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QSlider" name="xSlider">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>550</y>
     <width>701</width>
     <height>16</height>
    </rect>
   </property>
   <property name="maximum">
    <number>360</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="ySlider">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>600</y>
     <width>701</width>
     <height>16</height>
    </rect>
   </property>
   <property name="maximum">
    <number>360</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="zSlider">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>650</y>
     <width>701</width>
     <height>16</height>
    </rect>
   </property>
   <property name="maximum">
    <number>360</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="OpenGL_Class" name="OGL_Widget" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>641</width>
     <height>471</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>OpenGL_Class</class>
   <extends>QWidget</extends>
   <header location="global">opengl_class.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
